controllers/operator/mongodbsearch_controller.go

```diff
@@ -89,5 +90,6 @@ func AddMongoDBSearchController(ctx context.Context, mgr manager.Manager, operat
 		For(&searchv1.MongoDBSearch{}).
 		Watches(&mdbcv1.MongoDBCommunity{}, r.mdbcWatcher).
 		Owns(&appsv1.StatefulSet{}).
+		Owns(&corev1.Secret{}).
```

Comment:
just to ensure - we won't take any ownership of tls certificates referenced in the CR? I.e. those won't be deleted with deleting the CR?

Reply:
Correct. This is for the secret the operator creates which contains the concatenated certificate and key, just like the momgod secret.